osu.Game/Overlays/Notifications/ProgressNotification.cs

```diff
@@ -166,6 +165,8 @@ private void attemptPostCompletion()
 
             CompletionTarget.Invoke(CreateCompletionNotification());
             completionSent = true;
+
+            Close(false);
```

Comment:
Close applies transforms while `attemptPostCompletion` may get called from a non-update thread (database/network-related notifications I think), so a mutation exception may potentially be thrown? But scheduling this call defeats the point since it means the count would not update until the user opens the overlay. Perhaps the transform logic inside `Close` should be scheduled instead?

Reply:
Where are you seeing this?

Reply:
Just from reading code diff alone, haven't deeply checked if that's the case but it mostly seems to be.

Reply:
It should be scheduled, I can't see where it's not.

Reply:
Wow, I was checking the `Notification` class from master branch while comparing with the code changed here, without realising that the `Close` logic there was actually changed in this PR... Disregard this 😅